lib/Common/CommonDefines.h

```diff
@@ -140,7 +140,11 @@
 #define ENABLE_JS_ETW                               // ETW support
 #else
 #define SYSINFO_IMAGE_BASE_AVAILABLE 0
+#ifndef ENABLE_VALGRIND
 #define ENABLE_CONCURRENT_GC 1
+#else
+#define ENABLE_CONCURRENT_GC 0
```

Comment:
For the stack addresses, we do two approaches to mitigate this- we try to map in the write barrier cards corresponding to the script threads stack up front, and we also try to be disciplined about not setting barriers on writes to stack addresses (they're unnecessary). Still, @obastemur I think you're right that the 32GB approach might not work if under valgrind doesn't guarantee that the memory range is contiguous and starts from 0 (if it is contiguous and we know the start range, we can make the smaller card table idea work)